website/docs/r/policy_set.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_sentinel_policy"
+sidebar_current: "docs-resource-tfe-sentinel-policy"
+description: |-
+  Sentinel Policy as Code is an embedded policy as code framework integrated with Terraform Enterprise.
+---
+
+# tfe_policy_set
+
+Sentinel Policy as Code is an embedded policy as code framework integrated
+with Terraform Enterprise.
+
+Policy sets are configured on a per-organization level, and allow organization
+owners to choose which policies are enforced on which workspaces.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "tfe_policy_set" "test" {
+  name = "my-policy-set"
+  description = "A brand new policy set"
+  global = false
+  organization = "my-org-name"
+
```

Comment:
Is this blank line on purpose? Usually we only add blank lines when we have a configuration block, so it shouldn't be needed here.